splash/lua_modules/splash.lua

```diff
@@ -25,7 +24,7 @@ end
 -- It is required to handle errors properly.
 --
 function Splash:jsfunc(...)
-  local func = Splash_private.jsfunc(self, ...)
+  local func = self.__private.jsfunc(self, ...)
```

Comment:
In that case we should fix `drops_self_argument` because it make useless the usage of self.